milli/src/search/query_tree.rs

```diff
@@ -736,7 +747,11 @@ where
                 // 2. if the word is not the last token of the query and is not a stop_word we push it as a non-prefix word,
                 // 3. if the word is the last token of the query we push it as a prefix word.
                 if quoted {
-                    phrase.push(token.lemma().to_string());
+                    if stop_words.as_ref().map_or(false, |swords| swords.contains(token.lemma())) {
```

Comment:
I did this because it's used in the condition that was here before: https://github.com/meilisearch/milli/blob/b34fbcf4ebb2144e538f774dcb67d95581d66f6e/milli/src/search/query_tree.rs#L756

Should I change that too?

Reply:
Yes! Let's avoid searching for things directly accessible elsewhere! Thanks!

Reply:
With this, the `stop_words` parameter to this function (`create_primitive_query`) becomes useless! I'll remove it

Reply:
Done in 668241e8641035d0976856f5ba2356c712234140